### DISCUSSION TITLE
How to ensure the picker popup is visible inside other constructs

### DISCUSSION THREAD

**Discussion by cboger77** [mskocik/svelty-picker#133] [Q&A]
I downloaded the latest version of svelty-picker and suddenly the picker pop-up is restricted to the element that contains it, which make a lot of it invisible.

 <Container>
      <div class="card shadow-xl flex-grow">
        <div class="card-body">
          
          <a class="text-center text-lg font-bold mb-1">Remote Nursing Encounters Summary/Review</a>
          
           <center>
            <SveltyPicker
            inputClasses="form-control z-50 text-center text-sm" 
            title="Choose Time Range"
            manualInput = true
            isRange =true
            bind:value={timeRange}
            on:change={newEncounterRange}
          />

**Comment by mskocik** [comment]
And is the component wrapped by HTML element with `overflow: hidden`?
